modules/logging-rn-5.5.14.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+// cluster-logging-release-notes.adoc
+// logging-5-5-release-notes.adoc
+:_content-type: REFERENCE
+[id="cluster-logging-release-notes-5-5-14_{context}"]
+= Logging 5.5.14
+This release includes link:https://access.redhat.com/errata/RHBA-2023:4343[OpenShift Logging Bug Fix Release 5.5.14].
+
+[id="openshift-logging-5-5-14-bug-fixes_{context}"]
+== Bug fixes
+* Before this update, the Vector collector occasionally panicked with the following error message in its log: `thread 'vector-worker' panicked at 'all branches are disabled and there is no else branch', src/kubernetes/reflector.rs:26:9`. With this update, the error has been resolved. (link:https://issues.redhat.com/browse/LOG-4279[LOG-4279])
```

Comment:
"been resolved" = passive voice

Reply:
@briandooley I have fixed the passive voice comment and also the issue of 5.6.9 not showing. PTAL.